cmake/public/LoadHIP.cmake

```diff
@@ -94,12 +94,6 @@ ELSE()
   SET(MIOPEN_PATH $ENV{MIOPEN_PATH})
 ENDIF()
 
-IF(NOT DEFINED ENV{HCC_AMDGPU_TARGET})
-  SET(HCC_AMDGPU_TARGET gfx900;gfx906)
-ELSE()
```

Comment:
We can add gfx803 to the default list, but I would still prefer to keep this user-configurable.